csrc/anemone_mempool.c

```diff
@@ -48,6 +49,8 @@ void anemone_mempool_add_block (anemone_mempool_t *pool)
 // This happens when there is not enough space in the current block for the requested bytes.
 void * anemone_mempool_alloc_block (anemone_mempool_t *pool, size_t num_bytes)
 {
+  num_bytes = round_up_allocation(num_bytes);
```

Comment:
I think it's fine to treat it as a private function since the comment says "This is for internal use only, but must be exposed for inlining."